packages/angular_devkit/build_angular/src/browser/index.ts

```diff
@@ -141,6 +141,9 @@ export async function buildBrowserWebpackConfigFromContext(
     wco.buildOptions.webWorkerTsConfig ? getWorkerConfig(wco) : {},
   ];
 
+  // use the value for preserveSymlinks from NodeJS if it's not set in the config
+  options.preserveSymlinks = options.preserveSymlinks === undefined ? process.execArgv.includes('--preserve-symlinks') : options.preserveSymlinks;
```

Comment:
I think if we do this it should be somewhere else and effect all architect builders and not just this one. As this will result in inconsistencies between different builders.

Reply:
Ah dam, I thought this function was used by all builders. Ideas on where I should be looking I stead?

Reply:
https://github.com/angular/angular-cli/blob/30df1470a0f18989db336d50b55a79021ab64c85/packages/angular_devkit/build_angular/src/utils/normalize-builder-schema.ts#L50-L72

Also, in the various schemas you'd need to remove the default, since there is a default of `false`. `options.preserveSymlinks` it will never be `undefined`.

That being said, let's ping @clydin and double check that he's onboard with this change.